File: Course_01/C Language/EstruturaRepetitiva/soma_impares.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int x, y, menor, maior, inicial, soma;

    printf("Digite dois numeros:\n");
    scanf("%d", &x);
    scanf("%d", &y);

    if (x < y) {
        menor = x;
        maior = y;
    }
    else {
        menor = y;
        maior = x;
    }
    if (menor % 2 == 0) {
        inicial = menor + 1;
        }
    else {
        inicial = menor + 2;
    }

    soma = 0;

    while (inicial < maior) {
        soma = soma + inicial;
        inicial = inicial + 2;
    }
    printf("SOMA DOS IMPARES = %d", soma);

    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/soma_impares.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class soma_impares {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int x, y, temp, soma;
		
		System.out.println("Digite dois numeros: ");
		x = sc.nextInt();
		y = sc.nextInt();
		
		if (x > y) {
			temp = x;
			x = y;
			y = temp;
		}
		
		if (x % 2 == 0) {
			x += 1;
		}
		else {
			x += 2;
		}
		
		soma = 0;
		
		for (int i =  x; i < y; i += 2 ) {
			soma += i;
		}
		
		System.out.println("SOMA DOS IMPARES = " + soma);
		
		sc.close();
	}

}
